src/Mod/Assembly/Gui/ViewProviderAssembly.cpp

```diff
@@ -304,6 +317,14 @@ void ViewProviderAssembly::unsetEdit(int mode)
                                 "Gui.getDocument(appDoc).ActiveView.setActiveObject('%s', None)",
                                 this->getObject()->getDocument()->getName(),
                                 PARTKEY);
+
+        Gui::TaskView::TaskView* taskView = Gui::Control().taskPanel();
+        if (taskView) {
+            // taskView->removeContextualPanel(taskSolver);
```

Comment:
this could use comment too

Reply:
yeah, sometimes stuff happens and stuff like that is left in the code and after few years you have no idea why it is there